site/e2e/tests/webTerminal.spec.ts

```diff
@@ -28,13 +33,13 @@ test("web terminal", async ({ context, page }) => {
     ],
   })
   await createWorkspace(page, template)
-  await startAgent(page, token)
+  const agent = await startAgent(page, token)
```

Comment:
We still have the unfortunate problem that any error in a test will terminate execution and the agent won't be cleaned up. Can we somehow make it a post-test cleanup hook that always runs and checks if an agent is running?

This isn't a big problem per-se but can cause trouble when running locally -> suddenly all tests fail after one failure. And also cause noise in test output.